plugins/module_utils/common/xml_to_json.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+"""
+The index_of plugin common code
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import xmltodict
+import json
+from ansible_collections.ansible.utils.plugins.module_utils.common.utils import (
+    validate_data,
+)
+
+# Note, this file can only be used on the control node
+# where ansible is installed
+# limit imports to filter and lookup plugins
+try:
```

Comment:
Not required when moved under `plugin_utils` folder

Reply:
ack